README.md

```diff
@@ -1,2 +1,20 @@
 # python3-revpi-hat
+
 Python3 library for interacting with the RevPi HAT EEPROM
+
+## Example usage
+
+```python
+
+import revpi_hat
```

Comment:
Does the import revpi_hat here means all classes defined under directory revpi_hat/,  which make it works the same as from revpi_hat import RevPiHatEEPROM, RevPiHatEEPROMAttributeException in Example.

Reply:
with `import revpi_hat` you have to use the classes like `revpi_hat.RevPiHatEEPROM()`

with `from revpi_hat import RevPiHatEEPROM, RevPiHatEEPROMAttributeException ` you can use the classes like `RevPiHatEEPROM()`

For more examples see eg. https://www.codingem.com/python-difference-between-import-and-from-import/